index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>my portfolio</title>
+    <link rel="stylesheet" href="styling.css">
+  </head>
+  <body>
+    <div class="container">
+      <section class="section1">
+        <header>
+          <nav class="navigation">
+            <section class="logo">
+              <a href="index.html" class="logo-text touch rough">my logo</a>
+            </section>
```

Comment:
- [ ] Kindly 😃 try not to put the logo inside the section tag because the section tag is not used here, so please try to read about the usage of semantics [here](https://www.w3schools.com/html/html5_semantic_elements.asp) to get more information 💯

| The section tag| 
| --- |
|    ![image](https://user-images.githubusercontent.com/83360636/227517776-33e04672-c2d7-4fc5-917b-ad2aed695620.png)  |